games/TheBattleForWesnoth/Dockerfile

```diff
@@ -2,10 +2,11 @@ FROM  --platform=$TARGETOS/$TARGETARCH ubuntu:22.04
 
 LABEL author="Michael Parker" maintainer="parker@pterodactyl.io"
 
-ENV   DEBIAN_FRONTEND noninteractive
```

Comment:
leaves this, but with:

ARG   DEBIAN_FRONTEND noninteractive

Reply:
I think I did this but did I need both lines or just the one you have suggested?